xml/System.Windows.Forms/TableLayoutPanel.xml

```diff
@@ -247,6 +246,7 @@
   
  ]]></format>
         </remarks>
+        <exception cref="T:System.ComponentModel.InvalidEnumArgumentException">.NET 6 and later versions only: The property value is not valid for the enumeration type.</exception>
```

Comment:
I didn't find CellBorderStyle in TableLayoutSettings, so added the exception here.

Reply:
`TableLayoutSettings.CellBorderStyle` is internal.